presto-cli/src/main/java/com/facebook/presto/cli/Console.java

```diff
@@ -235,8 +236,14 @@ private static void runConsole(QueryRunner queryRunner, ClientSession session, A
 
                         process(queryRunner, split.statement(), outputFormat, true);
                     }
-                    reader.getHistory().add(squeezeStatement(split.statement()) + split.terminator());
+                    if (statements.length() == 0) {
+                        statements.append(squeezeStatement(split.statement())).append(split.terminator());
+                    }
+                    else {
+                        statements.append(" ").append(squeezeStatement(split.statement())).append(split.terminator());
```

Comment:
append a space can make the statements recall more visual friendly, if no space appended, the result would be:
"select 1;select 2;"
Now the result is "select 1; select 2;", there is a space between two statements.
What do you think?